velox/exec/fuzzer/AggregationFuzzerBase.cpp

```diff
@@ -319,7 +330,17 @@ std::vector<RowVectorPtr> AggregationFuzzerBase::generateInputDataWithRowNumber(
     }
 
     for (auto i = children.size(); i < types.size() - 1; ++i) {
-      children.push_back(vectorFuzzer_.fuzz(types[i], size));
+      if (partitionKeySet.find(names[i]) != partitionKeySet.end()) {
+        // The partition keys are built with a dictionary over a smaller set of
+        // values. This is done to introduce some repetition of key values for
+        // windowing.
+        auto baseVector = vectorFuzzer_.fuzz(types[i], numPartitions);
+        auto nulls = vectorFuzzer_.fuzzNulls(size);
+        children.push_back(
+            BaseVector::wrapInDictionary(nulls, indices, size, baseVector));
```

Comment:
Thanks for the feedback @kagamiori, `fuzzDictionary` fuzzes the dictionary indices each time and we want to use the same dictionary indices for all columns belonging to `partitionKeySet`, in order to ensure the rows with the same values for all `partitionKeySet` columns belong to the same partition.
Could you please share if this is fine?

Reply:
Yeah, that makes sense. Then you can move `auto indices = vectorFuzzer_.fuzzIndices(size, numPartitions)` inside the loop of `for (auto j = 0; j < FLAGS_num_batches; ++j)` since all batches do not have to use the same indices.

Reply:
Updated, thanks.